community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/Configuration.java

```diff
@@ -19,6 +19,12 @@
  */
 package org.neo4j.unsafe.impl.batchimport;
 
+import org.apache.commons.lang3.SystemUtils;
+import org.apache.lucene.util.IOUtils;
```

Comment:
I was torn between this, and building my own version of the same functionality as a utility method somewhere in the io module. You favour the latter approach?